### PR TITLE
Add Gelma's typos from "b" to "cl"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8165,6 +8200,7 @@ broswed->browsed
 broswer->browser
 broswers->browsers
 broswing->browsing
+brough->brought, borough, bough,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:8203]: No typos of `trough`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8378,6 +8418,7 @@ butttons->buttons
 buufers->buffers
 buuild->build
 buuilds->builds
+buyed->bought
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:8421]:
```suggestion
buyed->bought, buyer,
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -10261,6 +10359,8 @@ cheecksum->checksum
 cheecksums->checksums
 cheeper->cheaper
 cheeta->cheetah
+cheetsheet->cheatsheet
+cheetsheets->cheatsheets
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:10363]: Perhaps add `cheet`/`cheets`/`cheeted`/`cheeting` as typos of `cheat`, `sheet`, `cheek`, `cheep`, cheer, etc.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:8203]: I'll add `trough`

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**